Gemfile

```diff
@@ -4,6 +4,7 @@ group :test do
   gem 'pry-byebug'
   gem 'therubyracer'
   gem 'activeadmin', git: 'https://github.com/activeadmin/activeadmin.git'
+  gem 'rails-settings-cached', '~> 0.6.6'
```

Comment:
Why it should be in gemfile ? And i think we can remove `activeadmin` from this file because `activeadmin` has stable version